### ISSUE TITLE
Add /mods and /banned commands to list all modded and banned users. 

### ISSUE THREAD

**Issue by MCProhosting-Matt** [WatchBeam/widgets#19]

**Comment by JamyDev** [comment]
This should be done via the Beam API (`/api/v1/chats/:id/viewers` should have a role based filter)

**Comment by misterrick** [comment]
Whoa, sorry, I had no idea this issue was assigned to me, or even existed. For some reason, widgets is not included in ZenHub :/

**Comment by misterrick** [comment]
@JamyDev I can only see an `api/v1/chats/:id/users` endpoint on the backend, and it doesn't appear to have any kind of filtering, from what I can see in the code. Please correct me if I'm wrong, though.

We can do the filtering on the frontend if needed, at least the same request would be shared and cached for the viewer list, and the /banned and /mods commands.

**Comment by JamyDev** [comment]
Yeah I was not sure about it, so I guess we have to add that filtering to the backend :)

I'll make an issue.
